doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2809,22 +2812,27 @@ Configure a {product-title} instance to act as a remote copy where data from the
 [[configuring-the-global-copy]]
 ===== Configuring the Global Copy
 
-Configure a {product-title} instance to act as the global copy from which data is replicated to remote copies. This procedure must only be performed on servers where the *Database Synchronization* role will be enabled. These settings must be completed before enabling that role.
+Configure a {product-title} instance to act as the global copy from which data is replicated to remote copies.
 
-. Navigate to menu:Settings[Configuration].
+image:centralized_admin_enable.png[Enable Centralized Administration]
```

Comment:
Maybe this image would be better if it showed the subscription menu expanded so that the enable option is visible.